build.sbt

```diff
@@ -185,6 +185,20 @@ lazy val twirl = http4sProject("twirl")
   .enablePlugins(SbtTwirl)
   .dependsOn(core % "compile;test->test")
 
+lazy val zipkin = http4sProject("zipkin")
+  .settings(
+    description := "Zipkin support for http4s"
+  )
+  .dependsOn(
+    core % "compile->compile;test->test",
+    theDsl % "compile->compile;test->test",
+    server % "compile->compile;test->test",
+    client % "compile->compile;test->test",
```

Comment:
We probably want to consider breaking this into zipkin-common, zipkin-client, and zipkin-server, or else anything that uses zipkin is going to end up with a medium-sized ball of mud.